federation/cmd/federation-controller-manager/app/controllermanager.go

```diff
@@ -166,15 +165,6 @@ func StartControllers(s *options.CMServer, restClientCfg *restclient.Config) err
 		}
 	}
 
-	if controllerEnabled(s.Controllers, serverResources, deploymentcontroller.ControllerName, deploymentcontroller.RequiredResources, true) {
-		glog.V(3).Infof("Loading client config for deployment controller %q", deploymentcontroller.UserAgentName)
-		deploymentClientset := federationclientset.NewForConfigOrDie(restclient.AddUserAgent(restClientCfg, deploymentcontroller.UserAgentName))
-		deploymentController := deploymentcontroller.NewDeploymentController(deploymentClientset)
-		glog.V(3).Infof("Running deployment controller")
-		// TODO: rename s.ConcurrentReplicaSetSyncs
-		go deploymentController.Run(s.ConcurrentReplicaSetSyncs, wait.NeverStop)
```

Comment:
It is no longer relevant: it could be reintroduced in a future PR, but I haven't seen any evidence that it's useful.

Removing it as a flag could be interesting, since it's a breaking change to remove it. It's probably not a huge deal, since I don't think most people are running the controller manager themselves, but it's something we do need to think about. It's functionally harmless to leave it, though it's misleading.